mermaid module not found when running tests

### Description

Hello, I just updated Mermaid to version 10.0.0 and now the tests in my app stopped working with the error `Cannot find module 'mermaid' from './file.ts'`. This was not occurring in previous versions (> 9.4.0). Everything works fine when running the app, this error is only happening on tests.

I did some investigation and found out that Jest actual error message is `Error: No known conditions for "." entry in "mermaid" package`. Adding the `require` entry, pointing to `"./dist/mermaid.esm.mjs"` in `package.json` solved the issue :)

Any specific reason to remove the `require` entry? Anyhow, I'm opening a PR adding it to `package.json`


### Steps to reproduce

1. Install `mermaid` in a React app
2. Create a file importing `mermaid` as follows: `import mermaid from 'mermaid'`
3. Create a `test` file for the previous created file
4. Run tests
5. See the error `Cannot find module 'mermaid' from './file.ts'`

### Screenshots

_No response_

### Code Sample

_No response_

### Setup

**Desktop**

- OS and Version: Mac
- Browser and Version: Chrome


### Additional Context

_No response_

--- a/packages/mermaid/package.json
+++ b/packages/mermaid/package.json
@@ -7,6 +7,7 @@
   "types": "./dist/mermaid.d.ts",
   "exports": {
     ".": {
+      "require": "./dist/mermaid.esm.mjs",
       "types": "./dist/mermaid.d.ts",
       "import": "./dist/mermaid.core.mjs"
     },

--- a/packages/mermaid/package.json
+++ b/packages/mermaid/package.json
@@ -7,9 +7,9 @@
   "types": "./dist/mermaid.d.ts",
   "exports": {
     ".": {
-      "require": "./dist/mermaid.esm.mjs",
       "types": "./dist/mermaid.d.ts",
-      "import": "./dist/mermaid.core.mjs"
+      "import": "./dist/mermaid.core.mjs",
+      "default": "./dist/mermaid.core.mjs"
     },
     "./*": "./*"
   },

